src/views/store/Index.vue

```diff
@@ -1,81 +1,144 @@
 <template>
-    <Searchbar 
-    :categories="categories" 
-    :categorySelected="categorySelected"
-    :onChangeCategory="onChangeCategory"
-    :onChangeName="filterByName"
-    :search="search"
-    :height=" windowWidth < 1024 ? '48' : '80'"
-    />
+  <div class="index">
+    <Header />
+    <div class="indexMainSection">
+      <Searchbar
+        :categories="categories"
+        :categorySelected="categorySelected"
+        :onChangeCategory="onChangeCategory"
+        :onChangeName="filterByName"
+        :search="search"
+        :height="windowWidth < 1024 ? '48' : '80'"
+      />
+      <div v-if="categorySelected == 'TODOS'" class="products">
+        <Product
+          v-for="i in 3"
+          :key="i"
+          :category="categories[i]"
+          :products="products.filter((product) => product.category === categories[i])"
+        />
+      </div>
+      <Product
+        v-else
+        :category="categorySelected"
+        :products="products.filter((product) => product.category === categorySelected)"
```

Comment:
In the filteredProducts variable of the index component, I stored the products already filtered by the category selector, so you should use :products="filteredProducts" if not we are filtering twice.